java-manta-it/src/test/java/com/joyent/manta/http/TCPSocketConnectionTimeoutIT.java

```diff
@@ -27,9 +27,8 @@
  * Integration tests for verifying the connection timeout behaviour for an invalid connection.
  * This Integration test is written for the sole purpose of verifying Issue #441.
  *
- * @author <a href="https://github.com/dekobon">Ashwin A Nair</a>
```

Comment:
This is a good change, but it shouldn't be done as part of this PR.

Reply:
Sure.